### DISCUSSION TITLE
Ionos mail on Nextcloud

### DISCUSSION THREAD

**Discussion by clamyahoo** [the-djmaze/snappymail#1067] [Q&A]
Hello,
I try to use Snappymail on Nextcloud with an address from ionos. When preparing and testing it as an admin, the test shows ok, but if I try to use it later as a user, it shows a mistake saying that it can't connect to the host: because of143:
![grafik](https://user-images.githubusercontent.com/93087424/229525044-1471c5e4-ac4d-4338-ac82-2946e3b85c88.png)
With the normal Nextcloud mail app, it works. I've tried the connection with port 143 over telnet and it works. Anyway, like ionos says, the server access should be over SSL/TLS over port 993.
Does anybody have an idea what else I could do?
Kind regards,
Clemens

**Comment by cm-schl** [comment]
Hi,
maybe your nextcloud is using a normal username (not a email address) to log in to SnappyMail.
In this situation you must define a default domain (https://github.com/the-djmaze/snappymail/wiki/Admin-Manual#default-domain).
Otherwise you should activate the debug log and send us the specific part where the login happens 🙂
